### DISCUSSION TITLE
Can I chose the name of my pvc in destination ?

### DISCUSSION THREAD

**Discussion by mehdi-wsc** [backube/volsync#336] [Q&A]
Hello folks,
I'm using vsolync to replicate my PVCs between openshift clusters.
I need to keep my PVCs names but when I used volsync provisionner ( to create pvc) it generates random names, is there anyway to specify names of pvc with provisionner  ?
Regards

**Comment by JohnStrunk** [comment]
I'm going to assume you're talking about the PVC on the destination.

If you're using `copyMethod: Snapshot` on the destination (and you really should be, if possible), the PVC that VolSync creates is for its internal use, and it's named based on the name of the ReplicationDestination object. To get a PVC for your use, you can create a new PVC with whatever name you want and refer to the VolumeSnapshot (from the ReplicationDestination's `.status.latestImage`) in the new PVC's `.spec.dataSource` field.

If you're using `copyMethod: Direct` on the destination, you should create the destination PVC yourself (so it's owned by you and not VolSync). When creating the ReplicationDestination object, pass this PVC name as `.spec.*.destinationPVC` and it'll be used for incoming transfers instead of creating one.
